std/experimental/scripting.d

```diff
@@ -0,0 +1,83 @@
+/++
+Phobos is the D standard library, built on top of druntime.
+
+Convenience file that allows to import entire Phobos in one command.
++/
+module std.experimental.scripting;
```

Comment:
Just wondering, `scripting` seems to be a context-keyword. I understand, naming is a hard part, but do you think it's possible to rename?

Let's say down the line, the import times get much sleeker and people start to use this by default in D, `import std.scripting` would look odd  😉

Reply:
Andrei suggested `scripting`. My personal favorite would be `all`.
Other opinions?

Reply:
Also my hope is that if this gets accepted / popular, we can do `import std` (as initially proposed)

Reply:
`all` is what I came up to my mind as well and looks appropriate. Other options in the order of sensibility can be: `everything`, `completely`, `phobos`, `entirely`, `totally`. Can't think of other synonyms of `all`. 😄

Reply:
What about `std.experimental.allphobos`? I.e., import all of Phobos. :-P

I don't think the name matters that much, since this is `std.experimental`. Once it becomes widely adopted, people will just change that to `import std;` anyway.